incubator/command-dump/src/main/java/io/aklivity/zilla/runtime/command/dump/internal/airline/ZillaDumpCommand.java

```diff
@@ -0,0 +1,222 @@
+/*
+ * Copyright 2021-2022 Aklivity Inc
+ *
+ * Licensed under the Aklivity Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ *   https://www.aklivity.io/aklivity-community-license/
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package io.aklivity.zilla.runtime.command.dump.internal.airline;
+
+import static io.aklivity.zilla.runtime.engine.EngineConfiguration.ENGINE_DIRECTORY;
+import static java.lang.Integer.parseInt;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+
+import java.io.IOException;
+import java.io.RandomAccessFile;
+import java.net.URI;
+import java.nio.channels.FileChannel;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Properties;
+import java.util.function.Predicate;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+import org.agrona.LangUtil;
+import org.agrona.concurrent.BackoffIdleStrategy;
+import org.agrona.concurrent.IdleStrategy;
+
+import com.github.rvesse.airline.annotations.Command;
+import com.github.rvesse.airline.annotations.Option;
+
+import io.aklivity.zilla.runtime.command.ZillaCommand;
+import io.aklivity.zilla.runtime.command.dump.internal.airline.labels.LabelManager;
+import io.aklivity.zilla.runtime.command.dump.internal.airline.layouts.StreamsLayout;
+import io.aklivity.zilla.runtime.command.dump.internal.airline.spy.RingBufferSpy;
+import io.aklivity.zilla.runtime.engine.Configuration;
+import io.aklivity.zilla.runtime.engine.EngineConfiguration;
+
+@Command(name = "dump", description = "Dump stream content")
+public final class ZillaDumpCommand extends ZillaCommand
+{
+    @Option(name = {"-v", "--verbose"},
+        description = "Show verbose output")
+    public boolean verbose;
+
+    @Option(name = {"--version"})
+    public boolean version = false;
+
+    @Option(name = {"--continuous"})
+    public boolean continuous = true;
```

Comment:
Always continuous, no need for option.

Reply:
I only created this for the testing. I'll delete this option, and create a protected field for this. 

I still don't like it (creating a field that will only be used in test), but I can't think of any other quick solution, so we can terminate the process in the test.

Reply:
But I'm open for suggestion, if you have some idea